src/hotspot/cpu/x86/macroAssembler_x86.hpp

```diff
@@ -986,6 +986,16 @@ class MacroAssembler: public Assembler {
                  Register buf, Register state, Register ofs, Register limit, Register rsp,
                  bool multi_block);
 
+  // ChaCha20-Poly1305 macroAssembler defs
```

Comment:
These methods can also be moved to `stubGenerator_x86_64.hpp`/`stubGenerator_x86_64_chacha.cpp`. There are no other usages besides x86-64-specific CC20 stub.

Reply:
Done, and removed `macroAssembler_x86_chacha.cpp` since it is no longer needed.